### ISSUE TITLE
How can I set env for the tag in CI?

### ISSUE THREAD

**Issue by panli666** [bahmutov/cypress-select-tests#144]
I need to execute the cases in CI,  Can I use set 'CYPRESS_GREP' in buildkite yml file?

**Comment by bahmutov** [comment]
No, but it could be a new feature - either read the mask from CLI or from the environment variable
